packages/api/emails/decline/html.pug

```diff
@@ -0,0 +1,28 @@
+div.main
```

Comment:
I assume that this will render into `<div class="main">` however there isn't a defined css class of `main` injected to the email template.